pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -528,6 +528,7 @@ def test_offsets_hashable(self, offset_types):
 class TestDateOffset(Base):
     def setup_method(self, method):
         self.d = Timestamp(datetime(2008, 1, 2))
+        self.d2 = Timestamp("2021-10-01 08:00:00.000000000")
```

Comment:
is there a reason you need to add a new one? what's wrong with .d?

Reply:
Good point. I'll just reuse `d` then.